strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -566,12 +569,19 @@ def prepare_gkp(self, state, epsilon, ampl_cutoff, representation="real", shape=
         if representation == "complex":
             raise NotImplementedError("The complex description of GKP is not implemented")
 
-        if shape != "square":
-            raise NotImplementedError("Only square GKP are implemented for now")
+        if shape not in ["square", "rectangular"]:
+            raise NotImplementedError("Only square and rectangular GKP are implemented.")
+
+        if shape == "square":
+            if alpha != 1:
+                raise ValueError(
+                    "For square GKPs, alpha must be 1. For alpha not equal to "
+                    + "1, use shape='rectangular'."
+                )
 
         theta, phi = state[0], state[1]
 
-        def coeff(peak_loc):
+        def coeff(peak_loc, alpha):
```

Comment:
It's actually sqrt(alpha) that goes in here. So I would either keep it as is, and change the alpha in line 623 to sqrt(alpha) (and also change line 662), or instead call this something like beta for a bit more clarity. (This seems minor but might cause less confusion in further development)